webpack.config.dev.js

```diff
@@ -97,24 +82,31 @@ module.exports = {
     minimizer: [],
   },
   plugins: [
-    new HtmlWebpackPlugin({
-      inject: true,
-      hash: false,
-      template: './examples/index.html',
-      minify: {
-        removeComments: isProduction,
-        collapseWhitespace: isProduction,
-        removeAttributeQuotes: isProduction,
-        minifyJS: isProduction,
-        minifyCSS: isProduction,
-        minifyURLs: isProduction,
-      }
-    }),
+    new VueLoaderPlugin(),
     new webpack.DefinePlugin({
       __VUE_OPTIONS_API__: true,
       __VUE_PROD_DEVTOOLS__: true
     }),
-    new VueLoaderPlugin(),
+    new HtmlBundlerPlugin({
+      entry: {
+        'index': './examples/index.html',
```

Comment:
the syntax of the [entry](https://github.com/webdiscus/html-bundler-webpack-plugin#option-entry) plugin option is the same as original webpack `entry`.
The key of the entry object is the output file w/o `.html` extension.
For example:
```
'index': './examples/index.html', // => saves to "docs/index.html"
'about/index':  './examples/about.html', // => saves to "docs/about/index.html"
```